libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -125,8 +104,20 @@ add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-cmath-fp64.spv
                    DEPENDS device_math.h device.h clang llvm-spirv
                    VERBATIM)
 
+add_custom_target(libsycldevice-inc)
+add_custom_command(
+  TARGET libsycldevice-inc POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${include_dir}/CL/sycl/devicelib
+        COMMAND ${CMAKE_COMMAND} -E copy
```

Comment:
This looks ugly, but I don't know a better way to copy a file at build time.

Reply:
I believe this is the way to do it.  Clang headers are copied the same way, except that they use copy_if_different instead of copy.